src/docs/product/session-replay/replay-page-and-filters.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: "Replay Page and Filters"
+sidebar_order: 12
+description: "Session Replay page and filtering."
+---
+
+<Include name="beta-note-session-replay.mdx" />
+
+You can search for or browse replays of user sessions on the **Replay** page, where they’re  organized chronologically by default. You’ll see the following information for each session:
+
+- **Project**: The project in which the session was recorded.
+- **Start Time**: How long ago the replay was triggered. (Replays can appear here while the recording is still in progress.)
+- **Duration**: The length of the replay. (Recordings can range from 1 second to 60 minutes.)
+- **Errors:** The number of frontend errors identified by Sentry, as encountered in the replay. (Sometimes the error count reported is different from the actual errors captured. Read more in [Error Linking](/platforms/javascript/session-replay/#error-linking).)
+- **Activity**: The activity level as determined by the number of errors encountered, their duration, and the number of UI events.
+
+By default, the IP address is used to identify each replay on this page. This can be changed in your organization and project settings. If you’d like to set an email or a username instead, call [Sentry.setUser](/platforms/javascript/enriching-events/identify-user) in your client-side configuration.If you’ve enabled the option to “Prevent Storing of IP Addresses” in either your project-level or organization-wide settings, the IP address will appear redacted on the **Replay** page. For more information, read about [several data scrubbing options](/product/data-management-settings/scrubbing/server-side-scrubbing/) in the Sentry app on either an organization-wide or a project level.
```

Comment:
```suggestion
By default, the IP address is used to identify each replay on this page. This can be changed in your organization and project settings. If you’d like to set an email or a username instead, call [Sentry.setUser()](/platforms/javascript/enriching-events/identify-user) in your client-side configuration.If you’ve enabled the option to “Prevent Storing of IP Addresses” in either your project-level or organization-wide settings, the IP address will appear redacted on the **Replay** page. For more information, read about [several data scrubbing options](/product/data-management-settings/scrubbing/server-side-scrubbing/) in the Sentry app on either an organization-wide or a project level.
```

Reply:
LMK if that makes sense @bruno-garcia